### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_Alljobs_Screenreader:NVDA does not read the table along with the name of the column headers. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5915]
[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS4.2.3; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:Windows_NT x64 10.0.18362
**Additional Details:**
Tool Name: NVDA
Version:- 2019.1
**MAS Reference:**MAS1.3.1,MAS4.2.3
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the SQL agent tab->Jobs->Click on any the jobs in the table.
4.Navigate inside the table steps table.
5.Observe the announcement of NVDA
**Actual:**
Issue 1.NVDA is not announcing table along with the name of the column headers.
Issue 2.It is not  marked up as table.
Issue 3. NVDA is not announcing table name.

**Similar issue has been observed for below page -**

**1. NewJobAlerts -** 
Navigate to the SQL agent tab from Home page->Click on new job button->Select Alerts tab.

**Expected:**
NVDA should announce table data properly with the name of the column headers.
**User Impact:**
User with visiual disablities will find difficultto navigate through table.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.3.1,MAS4.2.3
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}

**Attachment for Reference:**

[5915 _AccessibilityA11y_AzureDataStudio_AllJobs_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3261477/5915._AccessibilityA11y_AzureDataStudio_AllJobs_Keyboard.pptx)
**Does this issue occur when all extensions are disabled?: Yes**
